builtin/providers/google/resource_container_cluster.go

```diff
@@ -444,7 +441,17 @@ func resourceContainerClusterRead(d *schema.ResourceData, meta interface{}) erro
 
 	d.Set("name", cluster.Name)
 	d.Set("zone", cluster.Zone)
-	d.Set("additional_zones", cluster.Locations)
+
+	if len(cluster.Locations) > 1 {
```

Comment:
It probably doesn't matter, but I'd love to set this in state, even if it's getting set to an empty value. I think, as a general rule of thumb, things should get set in state unless there's a core bug that makes things not work, we can articulate why diffing on it is undesirable, or we can articulate a strong security concern for storing it in state.